docs/troubleshooting.md

```diff
@@ -205,5 +205,23 @@ Override the default babel plugins to solve this issue. For example, leaving it
 ```
 
 Alternatively, you can start with the [default list defined in StrykerJS](https://github.com/stryker-mutator/stryker-js/blob/0c98f98649cecd7e7222cc5f168afbe1071099e5/packages/instrumenter/src/parsers/js-parser.ts#L8-L32) and start removing some of them until you get a working list.
-  
-  
+
+### Mutants survive but should be killed - Jest runner
+
+**Symptom**
+
+For a TypeScript file the mutants don't get killed but a coverage (in Jest) is shown.
+
+**Problem**
+
+The mutated source file is only implementing callbacks for a framework that uses express in the background as a server. The unit tests for this module are written with Jest and are using Axios to make REST calls against the server. There is no module dependency needed for the test. But Stryker is using Jest's `--findRelatedTests` to optimize the performance. And it looks like that this Jest feature is only looking at imports (see this [StackOverflow post](https://stackoverflow.com/questions/44066996/how-does-jest-findrelatedtests-work-under-the-hood).
+
+**Solution**
+
+Adding an (unused) import for the tested module get the mutants killed. This is not an optimal solution but it works.
```

Comment:
Maybe we can also mention that `--findRelatedTests` can also be disabled? https://stryker-mutator.io/docs/stryker-js/jest-runner#jestenablefindrelatedtests-boolean